apps/backend/core/src/test/java/com/schemafy/core/common/config/TestSecurityConfig.java

```diff
@@ -21,9 +55,13 @@ public SecurityWebFilterChain testSecurityFilterChain(
                 .httpBasic(ServerHttpSecurity.HttpBasicSpec::disable)
                 .formLogin(ServerHttpSecurity.FormLoginSpec::disable)
                 .logout(ServerHttpSecurity.LogoutSpec::disable)
+                .exceptionHandling(exceptionHandling -> exceptionHandling
+                        .authenticationEntryPoint(authenticationEntryPoint)
+                        .accessDeniedHandler(accessDeniedHandler))
                 .authorizeExchange(exchanges -> exchanges
-                        .pathMatchers(HttpMethod.OPTIONS).permitAll()
-                        .anyExchange().permitAll())
+                        .pathMatchers(PUBLIC_ENDPOINTS).permitAll()
```

Comment:
```suggestion
                        .pathMatchers(ApiPath.PUBLIC_API.replace("{version}", "**")).permitAll()
```

위 내용을 제거하고 이와 같은 형태로 공개 API 엔드포인트는 전부 열고, 현재 이외에 다른 엔드포인트가 추가적으로 사용되는 것은 없어서, 나중에 추가하는 것으로 하는게 좋을 것 같습니다.

Reply:
반영하겠습니다